CombinatorialGames/Game/Small.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2025 Tristan Figueroa-Reid. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tristan Figueroa-Reid
+-/
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Small games all around
+
+We define small games, games that are smaller than every Surreal
+and dicotic games, games `G` where both players can move from every nonempty subposition of `G`.
+
+## TODO
+
+- Define infinitesimal games as games x such that ∀ y : ℝ, 0 < y → -y < x ∧ x < y
+  - (Does this hold for small infinitesimal games?)
+- Prove that dicotic = small
```

Comment:
I'm not a fan of having two equivalent propositions, since that inevitably leads to API duplication. The Lawnmower theorem doesn't look too difficult to prove, perhaps we can have that as a first goal for this file?